resources/sts/4.14/sts_instance_controlplane_permission_policy.json

```diff
@@ -44,6 +44,17 @@
                 "kms:DescribeKey"
```

Comment:
I think we can remove the permission from here

Reply:
Ugh, force-pushed a fix